packages/worker/src/internals/MainThreadRunner.ts

```diff
@@ -41,7 +41,6 @@ export function runMainThread<Ts extends [unknown, ...unknown[]]>(
           return;
         }
         worker.register(predicateId, property.getPayload(inputs), resolve, reject, () =>
-          // eslint-disable-next-line @typescript-eslint/prefer-promise-reject-errors
```

Comment:
Actually this one needs to be added back. It seems that our CI check is not happy with this one being missing. I'm pretty sure that it's linked to the fact that our CI job does not rely on a pre-built version of fast-check. This it cannot know that a precondition is of type Error while it can know it when executed locally (usually you have the code built).

Thus I'd keep the disable comment for now.